lld/test/wasm/signature-mismatch.s

```diff
@@ -8,6 +8,11 @@
 # RUN: wasm-ld -r -o %t.reloc.o %t.main.o %t.ret32.o %t.call.o 2>&1 | FileCheck %s -check-prefix=WARN
 # RUN: obj2yaml %t.reloc.o | FileCheck %s -check-prefix=RELOC
 
+# RUN: rm -f %t.a
+# RUN: ar crS %t.a %t.ret32.o %t.call.o
```

Comment:
should this be `llvm-ar`? I wasn't sure this would work on e.g. Windows where there's no "system" ar, and the (few) other uses of 'ar' in tests use llvm-ar.

Reply:
Fixed by: https://github.com/llvm/llvm-project/commit/4b4763ffebaed9f1fee94b8ad5a1a450a9726683

Reply:
Thank you!